### PR TITLE
Make all log options available

### LINKED ISSUE #7: Only one log type can be chosen
Currently only one log type can be chosen.
```
log_publishing_options {
    enabled                  = "${var.log_publishing_enabled }"
    log_type                 = "${var.log_publishing_log_type}"
    cloudwatch_log_group_arn = "${var.log_publishing_cloudwatch_log_group_arn}"
}
```

It would be useful if more than one and optionally all 3 types can be enabled at once.

Maybe this way?
```
log_publishing_options {
    enabled                  = "${var.log_publishing_index_enabled }"
    log_type                 = "INDEX_SLOW_LOGS"
    cloudwatch_log_group_arn = "${var.log_publishing_index_cloudwatch_log_group_arn}"
}

log_publishing_options {
    enabled                  = "${var.log_publishing_search_enabled }"
    log_type                 = "SEARCH_SLOW_LOGS"
    cloudwatch_log_group_arn = "${var.log_publishing_search_cloudwatch_log_group_arn}"
}

log_publishing_options {
    enabled                  = "${var.log_publishing_application_enabled }"
    log_type                 = "ES_APPLICATION_LOGS"
    cloudwatch_log_group_arn = "${var.log_publishing_application_cloudwatch_log_group_arn}"
}
```

### PR DESCRIPTION
**Pull request by claaslisowski**
# What
* All available log options are individually configurable and can be enabled or disabled (default) 
* The cloudwatch log group arn can be reused for each logging type or a separate one can be used.

# Why
* To be able to choose any/all or none logging type instead of being bound to only one

Fixes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,9 +108,12 @@ Available targets:
 | instance_count | Number of data nodes in the cluster | string | `4` | no |
 | instance_type | Elasticsearch instance type for data nodes in the cluster | string | `t2.small.elasticsearch` | no |
 | kibana_subdomain_name | The name of the subdomain for Kibana in the DNS zone (_e.g._ `kibana`, `ui`, `ui-es`, `search-ui`, `kibana.elasticsearch`) | string | `kibana` | no |
-| log_publishing_cloudwatch_log_group_arn | ARN of the Cloudwatch log group to which log needs to be published | string | `` | no |
-| log_publishing_enabled | Specifies whether log publishing option is enabled or not | string | `false` | no |
-| log_publishing_log_type | A type of Elasticsearch log. Valid values: INDEX_SLOW_LOGS, SEARCH_SLOW_LOGS | string | `SEARCH_SLOW_LOGS` | no |
+| log_publishing_index_cloudwatch_log_group_arn | ARN of the Cloudwatch log group to which log needs to be published for INDEX_SLOW_LOGS | string | `` | no |
```
Comment (aknysh) [inline README.md]: can you change `Cloudwatch` everywhere to the canonical name `CloudWatch`?

**Comment by claaslisowski** [comment]
@aknysh,
thanks for the comment. 
I've changed all `Cloudwatch` to `CloudWatch`.
Tests have passed the check.

I've tested this config with some ES clusters.

**Comment by aknysh** [comment]
@claaslisowski 
can you also change to `CloudWatch` in README.yaml and then rebuild README by executing:

```
make init
make readme/deps
make readme
``

thank you

**Comment by aknysh** [comment]
ohh, you already did, sorry

**Pull request merged**